front-end/src/App.js

```diff
@@ -33,6 +34,14 @@ function App() {
         <Route path="/login">
           <Login />
         </Route>
+        <Route path="/editprofile" render={() => (
```

Comment:
<Login /> should probably be <Redirect to="/login" />.

Tested it locally and the URL doesnt change if you're not logged in. For example, remove access_token from localStorage and reload the page while in /dashboard. It shows the Login screen but still shows /dashboard up top.